components/nimbus/src/evaluator.rs

```diff
@@ -109,9 +111,15 @@ pub fn evaluate_enrollment(
 
 /// Check if an experiment is available for this app defined by this `AppContext`.
 ///
-/// `is_release` supports two modes:
-/// if `true`, available means available for enrollment: i.e. does the `app_name`, `app_id` and `channel` match.
-/// if `false`, available means available for testing: i.e. does only the `app_name` match.
+/// # Arguments:
+/// - `app_context` The application parameters to use for targeting purposes
+/// - `exp` The `Experiment` to evaluate
+/// - `is_release` Supports two modes:
+///     if `true`, available means available for enrollment: i.e. does the `app_name` and `channel` match.
+///     if `false`, available means available for testing: i.e. does only the `app_name` match.
```

Comment:
I'm still not sure about the naming of this: you correctly point out that `true` is for enrollment, `false` is for testing, so perhaps this is a simple `mode`, and we provide two further functions which call this one: 

```rust
fn is_experiment_available_for_testing(…) -> bool

fn is_experiment_available_for_enrollment(…) -> bool
```

Reply:
😬 I just fixed up the comment format to be compatible with rustdocs, the comment was pre-existing, as was the parameter and its usage.  I agree that the naming is a big confusing, fortunately this is internal to Nimbus and I'll file an issue to remind us to clear up this tech-debt.